.github/PULL_REQUEST_TEMPLATE/new_ea_pr_template.md

```diff
@@ -0,0 +1,22 @@
+## Description
+
+`<description>`
+
+## Steps to Test
+
+1. `yarn && yarn setup`
+2. `yarn test packages/<composites|sources|targets>/<adapter-name>/test`
+3. ...
+
+## New Adapter Checklist
+
+- [ ] Adapter was created by following [Creating a New Adapter](../../CONTRIBUTING.md#creating-a-new-adapter)
+- [ ] Endpoints are built following the style of the [example adapters](../../packages/examples).
+- [ ] Unit and integration tests cover all helper methods and endpoints.
+- [ ] Input/outputs are built to spec.
+- [ ] Code is documented with `const description = "..."` in each endpoint, and `description: "..."` in `schemas/env.json`.
+- [ ] Input parameters are defined in `const inputParameters = {...}` and `export type TInputParameters = {...}` in each endpoint file.
+- [ ] API credentials (ex. `API_KEY` and `API_SECRET` if applicable) are found in `schemas/env.json`.
```

Comment:
How about another line for _any_ environment variables should be found in schemas?

Reply:
Mind if I just modify this line instead of adding a new one? Trying to avoid making this too cluttered.

Reply:
Yeah that sounds good.